runelite-client/src/main/java/net/runelite/client/plugins/discord/DiscordGameEventType.java

```diff
@@ -262,7 +294,95 @@
+	REGION_BODY_ALTAR("Body Altar", DiscordAreaType.REGIONS, 10059),
+	REGION_CHAOS_ALTAR("Chaos Altar", DiscordAreaType.REGIONS, 9035),
+	REGION_COSMIC_ALTAR("Cosmic Altar", DiscordAreaType.REGIONS, 8523),
+	REGION_COSMIC_ENTITYS_PLANE("Cosmic Entity's Plane", DiscordAreaType.REGIONS, 8267),
+	REGION_CRABCLAW_ISLE("Crabclaw Isle", DiscordAreaType.REGIONS, 6965),
+	REGION_CRAFTING_GUILD("Crafting Guild", DiscordAreaType.REGIONS, 11571),
+	REGION_CRANDOR("Crandor", DiscordAreaType.REGIONS, 11314, 11315),
+	REGION_CRASH_ISLAND("Crash Island", DiscordAreaType.REGIONS, 11562),
+	REGION_DARK_ALTAR("Dark Altar", DiscordAreaType.REGIONS, 6716),
+	REGION_DEATH_ALTAR("Death Altar", DiscordAreaType.REGIONS, 8779),
+	REGION_DEATH_PLATEAU("Death Plateau", DiscordAreaType.REGIONS, 11320),
+	REGION_DENSE_ESSENCE("Dense Essence Mine", DiscordAreaType.REGIONS, 6972),
+	REGION_DRAGONTOOTH("Dragontooth Island", DiscordAreaType.REGIONS, 15159),
+	REGION_EAGLES_PEAK("Eagles' Peak", DiscordAreaType.REGIONS, 9270),
+	REGION_EARTH_ALTAR("Earth Altar", DiscordAreaType.REGIONS, 10571),
+	REGION_ENCHANTED_VALLEY("Enchanted Valley", DiscordAreaType.REGIONS, 12102),
+	REGION_FALADOR_FARM("Falador Farm", DiscordAreaType.REGIONS, 12083),
+	REGION_FARMING_GUILD("Farming Guild", DiscordAreaType.REGIONS, 4922),
+	REGION_FELDIP_HILLS("Feldip Hills", DiscordAreaType.REGIONS, 9773, 9774, 10029, 10030, 10285, 10286, 10287, 10542),
+	REGION_FIRE_ALTAR("Fire Altar", DiscordAreaType.REGIONS, 10315),
+	REGION_FISHER_REALM("Fisher Realm", DiscordAreaType.REGIONS, 10569),
+	REGION_FISHING_GUILD("Fishing Guild", DiscordAreaType.REGIONS, 10293),
+	REGION_FISHING_PLATFORM("Fishing Platform", DiscordAreaType.REGIONS, 11509),
+	REGION_FORSAKEN_TOWER("The Forsaken Tower", DiscordAreaType.REGIONS, 5435),
+	REGION_FOSSIL_ISLAND("Fossil Island", DiscordAreaType.REGIONS, 14394, 14395, 14649, 14650, 14651, 14652, 14906, 14907, 14908, 15162, 15163, 15164),
+	REGION_FREMENNIK("Fremennik Province", DiscordAreaType.REGIONS, 10296, 10552, 10808, 10809, 11064, 10810, 10811),
+	REGION_FREMENNIK_ISLES("Fremennik Isles", DiscordAreaType.REGIONS, 9276, 9532),
+	REGION_FROZEN_TUNDRAS("Frozen Tundras", DiscordAreaType.REGIONS, 6204, 6205, 6461, 6717),
+	REGION_GORAKS_PLANE("Gorak's Plane", DiscordAreaType.REGIONS, 12115),
+	REGION_GRAND_EXCHANGE("Grand Exchange", DiscordAreaType.REGIONS, 12598),
+	REGION_HARMONY("Harmony Island", DiscordAreaType.REGIONS, 15148),
+	REGION_ICE_PATH("Ice Path", DiscordAreaType.REGIONS, 11322, 11323, 11578),
+	REGION_ICEBERG("Iceberg", DiscordAreaType.REGIONS, 10558, 10559, 10658),
+	REGION_ISAFDAR("Isafdar", DiscordAreaType.REGIONS, 8497, 8498, 8753, 8754, 9009, 9010),
+	REGION_ISLAND_OF_STONE("Island of Stone", DiscordAreaType.REGIONS, 9790),
+	REGION_JIGGIG("Jiggig" , DiscordAreaType.REGIONS, 9775),
+	REGION_KANDARIN("Kandarin", DiscordAreaType.REGIONS, 9014, 9263, 9519, 9524, 9527, 9776, 9777, 9778, 9783, 10037, 10294, 10549, 10550, 10551, 10804, 10805, 10807),
```

Comment:
This should include 9264 since the only accessible part of that region is in Kandarin.
9777 should be the Observatory.
9778 should be the Ourania Hunter area.
Maybe 10549 should be labeled as the Ranging Guild?
10550 should be McGrubor's Wood.
10804 should be the Legends' Guild.
Maybe 10807 should be Sinclair Mansion?